fusion-plugin-rpc/src/server.js

```diff
@@ -114,19 +119,26 @@ const pluginFactory: () => RPCPluginType = () =>
     },
 
     middleware: deps => {
-      const {emitter, handlers, bodyParserOptions} = deps;
+      const {emitter, handlers, bodyParserOptions, rpcConfig} = deps;
       if (!handlers)
         throw new Error('Missing handlers registered to RPCHandlersToken');
       if (!emitter)
         throw new Error('Missing emitter registered to UniversalEventsToken');
       const parseBody = bodyparser(bodyParserOptions);
 
+      let apiPath = 'api';
```

Comment:
Perhaps we should add some logic to handle whether `rpcConfig.apiPath` was supplied with leading/trailing `/`. Not sure how the server handles adjacent `/`, so could lead to a confusing issue to debug.

```suggestion
      let apiPath = rpcConfig && rpcConfig.apiPath
        ? rpcConfig.apiPath
        : 'api';
      apiPath = `/${apiPath}/`.replace(/\/+/, '/');
```